examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/MainApplication.kt

```diff
@@ -13,38 +13,28 @@ import androidx.core.content.ContextCompat
 import com.google.android.material.dialog.MaterialAlertDialogBuilder
 import com.revenuecat.purchases.CustomerInfo
 import com.revenuecat.purchases.Purchases
-import com.revenuecat.purchases.PurchasesConfiguration
 import com.revenuecat.purchases.PurchasesError
-import com.revenuecat.purchases.amazon.AmazonConfiguration
 import com.revenuecat.purchases.interfaces.UpdatedCustomerInfoListener
+import com.revenuecat.purchases_sample.BuildConfig
 
 class MainApplication : Application(), UpdatedCustomerInfoListener {
 
     override fun onCreate() {
         super.onCreate()
-        StrictMode.setVmPolicy(
-            VmPolicy.Builder()
-                .detectLeakedClosableObjects()
-                .penaltyLog()
-                .penaltyDeath()
-                .build()
-        )
-        Purchases.debugLogsEnabled = true
 
-        val purchasesConfigurationBuilder =
-            when {
-                GOOGLE_API_KEY.isNotEmpty() -> PurchasesConfiguration.Builder(this, GOOGLE_API_KEY)
-                AMAZON_API_KEY.isNotEmpty() -> AmazonConfiguration.Builder(this, AMAZON_API_KEY)
-                else -> {
-                    throw IllegalArgumentException("Set at least one API key in Constants.")
-                }
-            }
-        Purchases.configure(purchasesConfigurationBuilder.build())
-        // set attributes to store additional, structured information for a user in RevenueCat.
-        // More info: https://docs.revenuecat.com/docs/user-attributes
-        Purchases.sharedInstance.setAttributes(mapOf("favorite_cat" to "garfield"))
+        if (BuildConfig.DEBUG) {
```

Comment:
I thought we probably don't want strict mode on the play store builds, which should be release builds.